Automattic/wp-calypso
# ETK: wpcom-block-description-support-link Add dispatch check before destructuring

Fixes: https://github.com/Automattic/wp-calypso/issues/87202

## Proposed Changes

Check if a dispatch function exists for `automattic/help-center` before using functions `setShowHelpCenter` and `setShowSupportDoc`.


Screenshots from Customizer widgets screen

Before:

<img width="297" alt="CleanShot 2024-02-05 at 18 17 20@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/1a8a0b1f-70cc-4f1d-a819-34934e1d9d43">


After:

<img width="301" alt="CleanShot 2024-02-05 at 18 19 28@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/a8e58004-4646-49fd-897d-0c4c5b232387">


## Testing Instructions



* Checkout branch
* Sandbox public.api and your simple site.
* Sync ETK to your sandbox (yarn dev --sync).
* For Atomic sites, deactivate the existing ETK plugin, upload the ETK plugin to your site from build below, and activate it.
* Verify Help Center is working on Calpyso, Site editor, and `Appearance > Customizer > widgets`
* Verify blocks are loading correctly when added in `Appearance > Customizer > widgets`


## Pre-merge Checklist
